docs/content/preview/yugabyte-voyager/known-issues/postgresql.md

```diff
@@ -1240,25 +1239,6 @@ EXECUTE FUNCTION check_and_modify_val();
 
 ---
 
-### Advisory locks is not yet implemented
```

Comment:
Has Voyager been updated to know that advisory locks are supported? And it is only so in 2.25.1, what if they migrate to a diffferent version?

Reply:
cc @priyanshi-yb

Reply:
Voyager changes are tracked by https://yugabyte.atlassian.net/browse/DB-13780. We probably should wait till that is done then.

Reply:
@ddhodge , Voyager changes are tracked by https://yugabyte.atlassian.net/browse/DB-13780. This update to the documentation is independant. Do you want the document changes to be done only after the Voyager changes land on master, so as to be consistent?

Reply:
Voyager uses the link to this section in assessment reports and we have a different workflow to mark a feature as fixed in certain versions, we have to do this separately once Voyager also picks the new YB version where Advisory lock is implemented. We generally don't remove the section in this docs, we just mention the YB version where the feature is fixed.

Reply:
Adding this back in so Voyager team can resolve from their end